test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -1,3 +1,10 @@
+##################################################################################################
+#
+# NOTE: IMPORTANT
+# ./test/dlc_tests/sanity/test_safety_check.py is obsolete and no longer used.
+# To create vulnerability exception, please declare them in ./data/ecr_scan_vulnerabilities.json
```

Comment:
Can we remove this second line. There is a new process that we will be shifting to, so its better not to mention this line here.